### DISCUSSION TITLE
Upgrade from leantime/leantime:2.1.4

### DISCUSSION THREAD

**Discussion by lexoyo** [Leantime/leantime#1194] [Q&A] [answered]
Hello

I try to migrate my existing install / DB from 2.1.4 to leantime/leantime:latest (as i understand this is [2.3.0-beta](https://hub.docker.com/layers/leantime/leantime/)

I keep the same DB and open /install/update, then I click the "upgrade" button

Then just a blank page with the warning "Warning: Undefined property: leantime\core\config::$plugins in /var/www/html/config/configuration.php on line 212"

Would someone know how to get this working?

Thank you

**Comment by marcelfolaron** [comment, marked as answer]
Hey,

Can you create an empty $plugin variable in your config file? Just a public
$plugin;

This should fix that. I have a fix for that ready for the next patch
release.

**Comment by marcelfolaron** [comment]
Do you see any errors in logs/error.logs ?

**Comment by lexoyo** [comment]
Nothing but the warning seems suspicious in the logs
When add `public $plugins;` to the class it works as expected ! thank you !!

**Comment by marcelfolaron** [comment]
Got it. I just released 2.3.1 which solves this in the config sample. You
can leave your file as is though.

**Comment by VINISHVV** [comment]
Hi,
I just updated to the latest beta version. It seems the profile image and company logo not updating.

**Reply by marcelfolaron** [reply]
Hi, 
thanks. We found the issue yesterday as well and will push a patch out today or tomorrow.
